scene/2d/node_2d.cpp

```diff
@@ -149,6 +149,14 @@ void Node2D::set_rotation(real_t p_radians) {
 	_update_transform();
 }
 
+void Node2D::set_rotation_degrees(real_t p_degrees) {
+	if (_xform_dirty) {
+		const_cast<Node2D *>(this)->_update_xform_values();
+	}
+	rotation = Math::deg_to_rad(p_degrees);
+	_update_transform();
```

Comment:
```suggestion
	set_rotation(Math::deg_to_rad(p_degrees));
```
This time there is no reason to not reuse it...

Reply:
yeah you are right, I thought one less function called may be good for performance, but I dont think it should make a difference

Reply:
I think compilers may optimize it to inline call.

Reply:
You missed a few methods :v

Reply:
I forgot to commit the changes, sorry :(